NomaiVR/Input/ActionInputs/ActionInputDefinitions.cs

```diff
@@ -0,0 +1,180 @@
+        {
+            SpecificAction = _default.StationaryUse
+        };
+
+        public static readonly Vector2ActionInput StationaryUp = new Vector2ActionInput(true, clamp: true)
+        {
+            SpecificAction = tools.DPad
+        };
+
+        public static readonly Vector2ActionInput StationaryDown = new Vector2ActionInput(true, true, true)
+        {
+            SpecificAction = tools.DPad
+        };
+
+        public static readonly Vector2ActionInput StationaryLeft = new Vector2ActionInput(invert: true, clamp: true)
+        {
+            SpecificAction = tools.DPad
+        };
+
+        public static readonly Vector2ActionInput StationaryRight = new Vector2ActionInput(clamp: true)
+        {
+            SpecificAction = tools.DPad
+        };
+
+        public static readonly SingleActionInput ThrustDown = new SingleActionInput
+        {
+            SpecificAction = _default.ThrustDown
+        };
+
+        public static readonly SingleActionInput ThrustUp = new SingleActionInput
+        {
+            SpecificAction = _default.ThrustUp
+        };
+
+        public static readonly Vector2ActionInput Up = new Vector2ActionInput(true, clamp: true)
+        {
+            SpecificAction = _default.UIDpad
+        };
+
+        public static readonly Vector2ActionInput Down = new Vector2ActionInput(true, true, true)
+        {
+            SpecificAction = _default.UIDpad
+        };
+
+        public static readonly Vector2ActionInput Left = new Vector2ActionInput(invert: true, clamp: true)
+        {
+            SpecificAction = _default.UIDpad
+        };
+
+        public static readonly Vector2ActionInput Right = new Vector2ActionInput(clamp: true)
+        {
+            SpecificAction = _default.UIDpad
+        };
+
+        public static readonly BooleanActionInput UISelect = new BooleanActionInput
+        {
+            SpecificAction = _default.UISelect
+        };
+
+        public static readonly BooleanActionInput UISubtabLeft = new BooleanActionInput
+        {
+            SpecificAction = _default.UISubtabLeft
+        };
+
+        public static readonly BooleanActionInput UISubtabRight = new BooleanActionInput
+        {
+            SpecificAction = _default.UISubtabRight
+        };
+
+        public static readonly BooleanActionInput UITabLeft = new BooleanActionInput
+        {
+            SpecificAction = _default.UITabLeft
+        };
+
+        public static readonly BooleanActionInput UITabRight = new BooleanActionInput
+        {
+            SpecificAction = _default.UITabRight
+        };
+
+        public static readonly BooleanActionInput ToolUse = new BooleanActionInput
+        {
+            SpecificAction = tools.Use,
+            IsEitherHand = true
+        };
+
+        public static readonly Vector2ActionInput ToolUp = new Vector2ActionInput(true, clamp: true)
+        {
+            SpecificAction = tools.DPad
```

Comment:
All the "Tool" actions are either hand interactions, apart from that seems all seems good.
Gonna test this whit the DLC content but it seems like it'll work just fine